Dockerfile

```diff
@@ -10,6 +10,12 @@ ARG STABLE_PG_VERSION=16
 ARG DEBIAN_VERSION=bookworm
 ARG DEBIAN_FLAVOR=${DEBIAN_VERSION}-slim
 
+ARG BOOKWORM_SLIM_SHA=sha256:40b107342c492725bc7aacbe93a49945445191ae364184a6d24fedb28172f6f7
```

Comment:
Please add comments:
- What are these SHAs? If I understand correctly, they're snapshots of what `debian:bookworm-slim:latest` tag points to today.
- When should these be updated? How to get the new SHA to use?

Reply:
they are `INDEX DIGEST`
To get them I went to the https://hub.docker.com/_/debian/tags?name=bookworm-slim, click on the tag name, and on next page it shows that `INDEX DIGEST`
- good part, that index contains all arches, so that sha is the same for all the arches, and that Dockerfile will work on all of them
- bad part: I didn't find how to get that SHA with cli tools
- related to update and to previous item: I will check how that index could be fetched with any SDK
- if it's a matter of few days, than I will build a tool, that will run by schedule, fetch current `INDEX DIGEST` and create PR with the update
- in the worst case, if no even SDK ways to do, than I will make some JS script, that will do the same

Reply:
Here is the script that will update pinned `sha`: https://github.com/neondatabase/cloud/pull/24174
After that PR merged, I will add that script and workflow to run it on a regular bases: if there are new versions, workflow will create PR with updates

Reply:
Ok, please add that information in a comment